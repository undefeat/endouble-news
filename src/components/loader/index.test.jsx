import React from 'react';
import shallow from 'react-test-renderer/shallow';
import Loader from '.';

describe('Loader', () => {
    it('should correctly render', () => {
        const tree = shallow.createRenderer().render(<Loader />);

        expect(tree).toMatchSnapshot();
    });
});
