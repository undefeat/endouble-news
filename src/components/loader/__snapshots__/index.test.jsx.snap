// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should correctly render 1`] = `
<div
  className="loader hidden"
>
  <div
    className="lds-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
